rskj-core/src/main/java/co/rsk/metrics/jmx/JmxMetricMBean.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * This file is part of RskJ
+ * Copyright (C) 2025 RSK Labs Ltd.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package co.rsk.metrics.jmx;
+
+import co.rsk.metrics.profilers.Profiler;
+
+import java.math.BigInteger;
+
+public interface JmxMetricMBean {
```

Comment:
Why do we need an interface?

Reply:
it's just a convention (more info [here](https://docs.oracle.com/javase/tutorial/jmx/mbeans/standard.html)). Also MBeanServer won't allow to register a "bean" concrete implementation without it being implementing such interface

Reply:
Good to know